mod/node-api/backend/backend.go

```diff
@@ -115,9 +116,10 @@ func New[
 		NodeT, StateStoreT, StorageBackendT, ValidatorT, ValidatorsT, WithdrawalT,
 		WithdrawalCredentialsT,
 	]{
-		sb: storageBackend,
-		cs: cs,
-		sp: sp,
+		sb:    storageBackend,
+		cs:    cs,
+		sp:    sp,
+		cache: cache.NewQueryCacheWithDefaultConfig(),
```

Comment:
if we're doing a whole config might be worthwhile to upstream these changes to an api config and have the api config divide into server + backend instead of server only which is what we have now, could be useful to be able to configure this